CHANGELOG.md

```diff
@@ -3,70 +3,17 @@ All notable changes to this project are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). See the [CONTRIBUTING guide](./CONTRIBUTING.md#Changelog) for instructions on how to add changelog entries.
 
-## [Unreleased 2.x]
+## [Unreleased 2.7]
```

Comment:
```suggestion
## [Unreleased 2.x]
```
?

Reply:
@reta do you want me to change this to `[Unreleased 2.x]` or are you fine with `[Unreleased 2.7]`?

Reply:
Oh sorry, I think `[Unreleased 2.7]` should be fine, I missed the convention here

Reply:
Thanks for confirmation.